tools/recetox_msfinder/macros.xml

```diff
@@ -0,0 +1,60 @@
+<macros>
+    <token name="@TOOL_VERSION@">0.1.0</token>
+    <xml name="requirements">
+        <requirements>
+            <container type="docker">recetox/recetox-msfinder:3.5.2-0</container>
+        </requirements>
+    </xml>
+    <xml name="creator">
+        <creator>
+            <person
+                givenName="Zargham"
+                familyName="Ahmad"
+                url="https://github.com/zargham-ahmad"
+                identifier="0000-0002-6096-224X" />
+            <organization
+                url="https://www.recetox.muni.cz/"
+                email="GalaxyToolsDevelopmentandDeployment@space.muni.cz"
+                name="RECETOX MUNI" />
+        </creator>
+    </xml>
+    <xml name="input">
+        <param name="input_data" type="data" format="msp,mat">
```

Comment:
```suggestion
        <param name="input_data" type="data" format="msp">
```

Do we really support `mat` at the moment? If so, there should be a test for it. I'd vote for rather not using this format.

Reply:
Also, it should be stated that Formula, InChI and SMILES codes are required to be present in the library file.

Reply:
resolved in 50fcbda